### PR TITLE
Improve clarity about how permissions are granted

### PR DESCRIPTION
**Pull request by benvolioT**
Improve clarity about how permissions are granted to users in more than one group, and for mapping files with overlapping iamPolicies.

### CODE REVIEW

**Comment by benvolioT** [comment]
Please do not merge this until @smayberry has a chance to review and approve.

**Comment by rackernexus** [comment]
Build 127 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-cc1ee52f6/docs/rackspace-federation/). valid for 14 days

**Comment by rackernexus** [comment]
Build 128 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-13cf139d4/docs/rackspace-federation/). valid for 14 days

File: doc/config-attribute-mapping/faws-mapping.rst [hunk 1]
```diff
@@ -162,6 +190,16 @@ policy below assigns permissions as follows:
               )
             multiValue: true
 
+In the above example, a user who is a member of both the
+``mycompany.global.security`` group and the ``mycompany.123456789012.admin``
+group, would be granted ``AdministratorAccess`` IAM policy for AWS account
+``123456789012``. This is because the full set of permissions granted to the
+user is the union of the permissions granted by each of the ``iamPolicies``
+in the ``aws`` section of the mapping YAML file.
```
Comment (smayberry) [inline doc/config-attribute-mapping/faws-mapping.rst:198]: Not entirely true... for policies it is always a union, there is no overriding or replacement.  So, in that example the user would get both `SecurityAudit` and `AdministratorAccess`.  But, so long as `SecurityAudit` does not have any explicit DENY statements, the effect is essentially `AdministratorAccess`.

File: doc/config-attribute-mapping/faws-mapping.rst [hunk 2]
```diff
@@ -93,6 +87,40 @@ policy, members of the ``mycompany.scoped.admin`` group are granted the FAWS
               )
             multiValue: true
 
+In the above example, a user who is a member of both the
+``mycompany.scoped.admin`` group and the ``mycompany.scoped.observer`` group,
+would be granted the FAWS ``admin`` role on the single account ``12345678012``.
+
+However, if the mapping order for the admin and observer groups were swapped as
```
Comment (benvolioT) [inline doc/config-attribute-mapping/faws-mapping.rst:94]: @smayberry  - is this part accurate?

File: doc/config-attribute-mapping/faws-mapping.rst (hunk 1 above)
Comment (benvolioT) [reply doc/config-attribute-mapping/faws-mapping.rst]: I believe I've corrected this.

**Comment by rackernexus** [comment]
Build 129 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-38b58cb19/docs/rackspace-federation/). valid for 14 days

File: doc/config-attribute-mapping/faws-mapping.rst (hunk 1 above)
Comment (benvolioT) [reply doc/config-attribute-mapping/faws-mapping.rst]: @smayberry  - can you please double check my change and confirm it looks okay to you?

File: doc/config-attribute-mapping/faws-mapping.rst (hunk 2 above)
Comment (smayberry) [reply doc/config-attribute-mapping/faws-mapping.rst:94]: Yes, I believe so.

File: doc/config-attribute-mapping/faws-mapping.rst (hunk 1 above)
Comment (smayberry) [reply doc/config-attribute-mapping/faws-mapping.rst]: Looks good to me.

**Comment by WilliamLoy** [comment]
@benvolioT What do IAM and ARN stand for? Thanks

**Comment by WilliamLoy** [comment]
@catlook ready for edits pending response on what those acronyms mean in my previous comment. 👯

**Comment by rackernexus** [comment]
Build 130 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-9c7b19422/docs/rackspace-federation/). valid for 14 days

**Comment by catlook** [comment]
on it!

**Comment by rackernexus** [comment]
Build 131 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-af863d61b/docs/rackspace-federation/). valid for 14 days

**Comment by WilliamLoy** [comment]
> @benvolioT What do IAM and ARN stand for? Thanks

@benvolioT

**Pull request merged**